### DISCUSSION TITLE
Helium Tax says No rewards during 2022

### DISCUSSION THREAD

**Discussion by pgiesselmann** [davetapley/helium-tax#92] [Q&A]
Hello,

I am trying to do my helium tax export. 

Website:
https://davetapley.com/helium-tax/

Wallet public adress:
14X6HSYqx9WPQdtRXiQhSHGqaTEhzRJvXeWi1QcSQFdXQwYS1oV

Script says:
No rewards during 2022

Can someone please help?

Thanks and greets

**Comment by dustinkeeton** [comment]
I'm having the same issue...
